main.go

```diff
@@ -45,7 +45,7 @@ func main() {
 		go runServer(shutdown)
 		go runMetricExporter()
 	}
-
+	logging.Log.Infof(conf.String_Integrate())
```

Comment:
If you call the above function `String()`, this call can end up being like this:

```diff
- logging.Log.Infof(conf.String_Integrate())
+logging.Log.Info(conf)
```

And the logger will call that `String()` function to print all the stuff.